backend/jottacloud/jottacloud.go

```diff
@@ -634,6 +638,15 @@ func (f *Fs) DirMove(src fs.Fs, srcRemote, dstRemote string) error {
 	return nil
 }
 
+// About gets quota information
+func (f *Fs) About() (*fs.Usage, error) {
+	usage := &fs.Usage{
```

Comment:
This is more useful if it is "live" so that a person using a jottacloud mount can run `df` and see the amount changing.  Mount caches the value for the same length of time as the directory cache time.